utils/action.py

```diff
@@ -0,0 +1,41 @@
+import torch
+import numpy as np
+from enum import Enum
+
+class Action(Enum):
+    """
+    From all actions we use only 3 in Pong
+    All actions:
+        0: NOOP
+        1: FIRE
+        2: RIGHT
+        3: LEFT
+        4: RIGHTFIRE
+        5: LEFTFIRE
+    """
```

Comment:
ACTION_MEANING = {    0: "NOOP",    1: "FIRE",    2: "UP",    3: "RIGHT",    4: "LEFT",    5: "DOWN",    6: "UPRIGHT",    7: "UPLEFT",    8: "DOWNRIGHT",    9: "DOWNLEFT",    10: "UPFIRE",    11: "RIGHTFIRE",    12: "LEFTFIRE",    13: "DOWNFIRE",    14: "UPRIGHTFIRE",    15: "UPLEFTFIRE",    16: "DOWNRIGHTFIRE",    17: "DOWNLEFTFIRE",} - https://github.com/openai/gym/blob/master/gym/envs/atari/atari_env.py, czyli nas interesuje 0,2,5 - testowane

Reply:
Tbh nie mam pojęcia czemu `env.unwrapped.get_action_meanings()` daje zły wynik

Reply:
To jest to samo

Reply:
Jak testowałem to 3 i 5 idą w dół, pewnie dlatego że dół i prawo robi to samo w tej grze

Reply:
Ale no lepiej mieć UP-DOWN